src/main/java/org/apache/ibatis/migration/CommandLine.java

```diff
@@ -24,8 +24,9 @@
 
 import org.apache.ibatis.migration.commands.Command;
 import org.apache.ibatis.migration.options.SelectedOptions;
+import org.apache.ibatis.migration.ConsoleColors;
 
-public class CommandLine {
+public class CommandLine implements ConsoleColors {
```

Comment:
This looks good @schumacher-m but CommandLine implementing ConsoleColors just to bring in the colors is a bit odd. Why not just static imports if you are that concerned about the colors.

Reply:
Well initially it was just a class I've imported (see: https://github.com/mybatis/migrations/pull/104/commits/6be27e95a008e50faff5f339a8ee43e1fe4a5cf6)

In the end I was indifferent about either way. What would you suggest?

Reply:
Well initially it was just a class I've imported (see: https://github.com/mybatis/migrations/pull/104/commits/6be27e95a008e50faff5f339a8ee43e1fe4a5cf6)

In the end I was indifferent about either way. What would you suggest?

Reply:
importing it makes more sense. the public static final are unnecessary anyways as interface values are already public final and static by default but you already made that change. So it's just adding the static imports or quantifying it.
In the future I would love to see the console writing be split out into a strategy so that you can use color console/logging as a strategy across not just the final error and status lines but this is a great feature on it's own.

Reply:
To clarify, I mean to give colors to warnings or pre/post hooks etc.

Reply:
Ok. Let me switch it back to just a class for now. We can continue on the output. I also like to make it less maven-esque (or at least define some sort of different formatter)

Reply:
There's some confusion here, I meant keep it an interface but don't implement it.